src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -212,6 +212,11 @@ private void ensureSelectedPersonIsValid(ListChangeListener.Change<? extends Per
         }
     }
 
+    @Override
+    public boolean checkNoCopy() {
```

Comment:
Do refer to the coding style document on naming of boolean variables to identify a name that might be more appropriate